sdk/core/core-crypto/README.md

```diff
@@ -0,0 +1,35 @@
+# Azure Core Crypto library for JavaScript (Experimental)
+
+This library is primarily intended to contain cryptographic helper functions for use by the Azure SDK for JavaScript.
+
+## Getting started
+
+### Requirements
+
+- [Node.js](https://nodejs.org) version > 8.x
+
+### Installation
+
+This package is not meant to be consumed directly by end users.
+
+## Key concepts
+
+TODO: ???
```

Comment:
for now a list of supported protocols might be sufficient here

Reply:
Updated!